cli/command_exec.go

```diff
@@ -84,12 +87,12 @@ func ExecCommand(cl *rokka.Client, options *CommandOptions, userArgs []string) e
 				return err
 			}
 
-			t, err := template.New("").Funcs(funcMap).Parse(tmpl + "\n")
+			t, err := template.New("").Funcs(funcMap).Parse(strings.TrimSuffix(tmpl, "\n") + "\n")
```

Comment:
why that?

Reply:
It's sometimes problematic to avoid trailing new lines in templates, e.g. when there's a loop at the end, you'd be better off avoiding trailing newlines by stripping them.

On the other hand, I agree that this is rather ugly.

Reply:
Added trailing newline to each template instead